core/wiki/macros/misc.tid

```diff
@@ -0,0 +1,263 @@
+title: $:/core/macros/misc
```

Comment:
IMO it should be `$:/core/procedures/utilities` or something like this.

Reply:
yeah probably not "misc"

Reply:
Now it's "utilities"